en/docs/administer/configuring-a-read-only-ldap-user-store.md

```diff
@@ -147,31 +141,41 @@ user store:
         <Property name="ConnectionName">uid=AdminLDAP,ou=system</Property>
         <Property name="ConnectionPassword">2010#Avrudu</Property>
         ```
+        ```toml
+        [user_store.properties]
+        "ConnectionName" = "uid=AdminLDAP,ou=system"
+        "ConnectionPassword" = "2010#Avrudu"
+        ```
 
-    3.  Update
-        `             <Property name="UserSearchBase">            ` with
+    3.  Update property
+        `             UserSearchBase       ` with
         the directory name where the users are stored. When LDAP
         searches for users, it will start from this location of the
         directory.
 
         ``` xml
         <Property name="UserSearchBase">ou=system</Property> 
         ```
+        ```toml
+        [user_store.properties]
+        "UserSearchBase" = "ou=system"
```

Comment:
Should these properties be within double quotes? If not, please change in all relevant places.